api/v3/Case/Calculatealldurations.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/**
+ * API call to calculate the duration of all cases.
+ */
+function civicrm_api3_case_calculatealldurations() {
+  $logger = new CRM_Civicase_CaseDurationLog();
+  $logger->calculateAllCasesDuration();
+
+  return array();

```

Comment:
Why the return array? Also why is this endpoint needed? Maybe you could add something about it in the PR description